pennylane/devices/default_qubit_autograd.py

```diff
@@ -69,6 +69,13 @@ class DefaultQubitAutograd(DefaultQubit):
 
     Args:
         wires (int): the number of wires to initialize the device with
+        shots (int): How many times the circuit should be evaluated (or sampled) to estimate
+            the expectation values. Defaults to 1000 if not specified.
+            If ``analytic == True``, then the number of shots is ignored
+            in the calculation of expectation values and variances, and only controls the number
+            of samples returned by ``sample``.
+        analytic (bool): indicates if the device should calculate expectations

```

Comment:
I can't remember. At one point during code review, `default.qubit.autograd` did not support non-analytic mode, since backprop is not well defined. It must have been added back for forward pass only, but the docstring not updated.

Reply:
@trbromley can you update the docstring to mention that non-analytic mode does not support `diff_method="backprop"`?

Reply:
And maybe even say that for non-analytic gradients, better to use `default.qubit` and `diff_method="parameter-shift"`, rather than this device.

Reply:
I've updated to this (also in default.qubit.tf): dcd3719af6f788a53d8ffc8269905150a47c9a49

Reply:
Thanks @trbromley!